File: benchmarks/runtime/bitwise_ops.lua
local n = 2000000
local i = 0
local sum = 0
while i < n do
  local a = i & 0xFF
  local b = a | 0x10
  local c = b ~ 0x55
  local d = c << 2
  local e = d >> 1
  sum = sum + e
  i = i + 1
end
if sum <= 0 then os.exit(1) end
print(sum)

File: benchmarks/runtime/bitwise_ops.js
let n = 2000000;
let i = 0;
let sum = 0;
while (i < n) {
  const a = i & 0xFF;
  const b = a | 0x10;
  const c = b ^ 0x55;
  const d = c << 2;
  const e = d >> 1;
  sum += e;
  i += 1;
}
if (sum <= 0) process.exit(1);
console.log(sum);
